### PR TITLE
[1.13.x-blue] update brew packages name, repositories names and branches

### PR DESCRIPTION
**Pull request by rgdoliveira**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO|RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: bamoe-kogito-builder-rhel8-overrides.yaml [hunk 1]
```diff
@@ -77,10 +77,10 @@ osbs:
           - ppc64le
       compose:
         pulp_repos: true
-  extra_dir: osbs-extra/rhpam-kogito-builder-rhel8
+  extra_dir: osbs-extra/ibm-bamoe-kogito-builder
```
Comment (rgdoliveira) [inline bamoe-kogito-builder-rhel8-overrides.yaml]: I'm not 100% sure about this one, should it be changed like I did?

Comment (desmax74) [reply bamoe-kogito-builder-rhel8-overrides.yaml]: Maybe ibm-bamoe-kogito-builder-rhel8

Comment (rgdoliveira) [reply bamoe-kogito-builder-rhel8-overrides.yaml]: yes, my bad, I removed the rhel8 by accident, I amended my previous PR

**Review by desmax74 — APPROVED** [review]

File: bamoe-kogito-builder-rhel8-overrides.yaml (hunk 1 above)
Comment (rgdoliveira) [reply bamoe-kogito-builder-rhel8-overrides.yaml]: @desmax74 
from OSBS documentation:
`If a directory name specified by extra_dir key will be found next to the image descriptor, this directory and its contents will be copied into the target directory and later to the dist-git directory`

Not sure if we had some kind of `rhpam-kogito-builder-rhel8` directory that was getting copied inside the target folder

**Comment by kie-ci1** [comment]
**PR job** `#18` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/18/display/redirect

**Test results:**
- PASSED: 41
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/18/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

**Review by spolti — APPROVED** [review]

**Pull request merged**